election-api-java/src/test/java/bbc/news/elections/ElectionsApiApplicationIntegrationTests.java

```diff
@@ -44,20 +48,42 @@ public void reset() throws Exception {
 		resultService.reset();
 	}
 
-
 	@Test
 	public void first5Test() throws Exception {
-		Scoreboard scoreboard = runTest(5);
+		JsonNode scoreboard = runTest(5);
 
 		assertNotNull(scoreboard);
 		// assert LD == 1
 		// assert LAB = 4
 		// assert winner = noone
+
+		// Some Jackson JSON API examples:
```

Comment:
This set of examples takes up a fair bit of space but I didn't want to steer the candidates towards a specific data structure (e.g. key-value object vs. array).